releases/point-2022-3.md

```diff
@@ -0,0 +1,81 @@
+
+
+## Improvements & Bug Fixes
+
+* **Emails sent by SysKit Point now have customized dynamic subject field**.
+     * The subject line shows the context of the email; for example, Communication site is inactive.
+* **Improved the loading speed** for Access Review Options, **improved the functionality** of the preview email and dialogue on **the Access Review Options page**.
+* **Improved performance** for Privileged Identity Management (PIM).
+    * Privileged Identity Management (PIM) allows some users to assign themselves Global Admin rights for a limited amount of time.
+    * Global Admin sync improved to occurs every 4 minutes. 
+    * Fixed a bug where AAD groups weren’t taken into account for PMI if some members of those groups had administrative roles assigned.
+* **Improved speed and performance** for the One Drive tile when accessing through User details.
+* **Optimized speed and performance** when loading the Lifecycle Management details screen.
+* **Improved speed** for loading the Overview tile when accessing the Private Channel details screen.
+* **Improved messaging** when a Bad Request or Internal server error occurs that provides more insight into the issue that occurred when more information is available.
+* **Improved Access Review feature** to include exact dates when accessing the PR History Details Overview.
+  * Names of columns changed to include Status, Performed On and Performed By.
+    * Status bar states whether the action was Completed or Expired, and hovering above it provides information on who completed the task.
+* **Improved speed** for accessing the Lifecycle Management Email Preview.
+* **Fixed a bug** where the New Update Available ribbon was showing even after updating SysKit Point to the newest version.
+* **Fixed a bug** where the loading screen was cut off generating the details report.
+    * Once loading was completed, the report would be unresponsive when trying to select lines.
+* **Fixed an issue** where an error message was shown after opening the Details screen on Alerts.
+* **Fixed an issue** where removing members from a Group or Team through Microsoft 365 portal doesn’t sync properly.
+    * When all members from a group or team are removed, and the group or team is empty after syncing, SysKit Point didn’t update that action and left all the members that were there from the previous sync.
+* **Fixed a bug** where workdays wouldn’t show when scheduling an Access Review using 2k resolution.
+    * When trying to schedule an Access Review and entering the number of workdays allowed to complete the request, the number wouldn’t show in the selection box when using 2k resolution (2560x1440).
+* **Fixed an issue** where a large quantity of deleted groups caused problems with the automatic sync function.
+    * The automatic sync would process groups, and if a tenant has more than 1000 deleted groups, upon seeing they are deleted, the sync action would be canceled, and the state reverted to idle.  
```

Comment:
'if a tenant had' ako se ne varam
također, umjesto da opisujemo statuse (reverted to idle) bolje je opisati koja je posljedica ovog issue-a - 
ne bih previše išao u dubinu issuea, pa predlažem sljedeće:

* **Fixed an issue** where the Group Sync would fail in environments with more than 1000 deleted Security Groups.